### ISSUE TITLE
Can't resize stage in dev mode

### ISSUE THREAD

**Issue by brollb** [jmoenig/Snap--Build-Your-Own-Blocks#1141]
If you try to resize the stage's panel (using the light grey handle on the left of the stage), it will move the entire snap world rather than the given panel

**Comment by jmoenig** [comment]
yes, that's actually kinda inevitable because of how resizers interact with the hand...
